invenio_records_rest/errors.py

```diff
@@ -187,3 +187,10 @@ def __init__(self, error=None, **kwargs):
         super(RESTValidationError, self).__init__(**kwargs)
         self.description = 'Validation error: {0}.'.format(
             error.message if error else '')
+
+
+class UnhandledElasticsearchError(RESTException):
+    """Failed to handle exception."""
+
+    code = 400
+    description = 'An exception occured but there was no handler for it.'
```

Comment:
This is displayed to users, so I would make it a bit more user-friendly in the wording. Is it correct that this error happens when a we execute a bad search query? If so, I would default to something like: ```Bad query syntax.```.

Alternatively, if the error is on our side, I would return a 500 error, with something like ``An internal server error happened during the search.``


```Failed to run